pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -139,6 +143,7 @@ func NewKubeGenericRuntimeManager(
 	runtimeService internalapi.RuntimeService,
 	imageService internalapi.ImageManagerService,
 	internalLifecycle cm.InternalContainerLifecycle,
+	dockerLegacyService dockershim.DockerLegacyService,
```

Comment:
can we pass in an interface scoped to what this provides?  for example, `LegacyLogProvider` or something?  this way, we don't need to pass the full legacy service through?